wfl/Database/MongoDBManager.cs

```diff
@@ -8,55 +7,31 @@
 
 namespace Symphono.Wfl.Database
 {
-    public class MongoDBManager: IDBManager
+    public class MongoDBManager<T>: IDBManager<T> where T: IEntity
     {
         private static string connectionString;
         public static MongoClient client;
         public static IMongoDatabase db;
+        private readonly string collectionName; //{ get; }
         
-        public MongoDBManager()
+        public MongoDBManager(string inputCollectionName, string inputConnectionString)
         {
-            connectionString = WebConfigurationManager.ConnectionStrings["databaseConnectionString"].ConnectionString;
+            connectionString = inputConnectionString;
             client = new MongoClient(connectionString);
```

Comment:
Given that we now are creating multiple Mongo DB Managers, I think we need to also change to where the MongoClient is a parameter that is passed into the DB Manager instead of passing the connection string.  Then we should use a single MongoClient for all MongoDBManagers.  Right now we're going to create many new connections.